comps/llms/rayllm/README.md

```diff
@@ -0,0 +1,48 @@
+# RayLLM Endpoint Service
```

Comment:
To differentiate this with Ray-LLM, we'd better change it to Ray-Serve.

Reply:
Thanks for your comments. Have updated it.